packages/keyda-demo/src/lib/Input/index.js

```diff
@@ -10,7 +10,7 @@ import _ from 'lodash';
 
 import { useKeyStateState, useKeyStateDispatch } from '../Context';
 
-const Input = (props) => {
+const Input = forwardRef((props, ref) => {
   const { type, onChange, value, ...rest } = props;
```

Comment:
<img width="868" alt="스크린샷 2020-05-20 오후 11 56 50" src="https://user-images.githubusercontent.com/14817195/82461603-95da5400-9af5-11ea-8d61-fc862ae21422.png">

I found a Warning like a picture up here.
To resolve this, I deleted `forwardRef` keyword transferring the component itself as a parameter  at the bottom and re-wrote them.